### PR TITLE
ICBM152 description: fmriprep vs nilearn

### LINKED ISSUE #3140: Add notes about ICBM template and the relationship with fMRIprep default template
It is not clear that the default template used by fMRPrep (https://github.com/templateflow/tpl-MNI152NLin2009cAsym) is the ICBM template.
This need to be make clear in the doc string.

### Affected pages

https://nilearn.github.io/modules/generated/nilearn.datasets.fetch_icbm152_2009.html#nilearn.datasets.fetch_icbm152_2009

Edit: wrong template flow link

### PR DESCRIPTION
**Pull request by htwangtw**
Closes #3140

Changes proposed in this pull request:

- Add a description about fmriprep and nilearn icmb version difference
- Add reference to templateflow

@bthirion suggested the wording of updating template, but I am not sure if this is relevant at this stage, and I feel we should encourage users to use templateflow for more specific version.

### CODE REVIEW

File: nilearn/datasets/struct.py [hunk 1]
```diff
@@ -46,6 +46,11 @@
 def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     """Download and load the ICBM152 template (dated 2009).
 
+    The default template of :term:`fMRIPrep` is the asymmetrical ICBM152 2009,
+    release c (MNI152NLin2009cSAsym). The NiLearn template is asymmetrical
+    ICBM152 2009, release a. If you wish to use the exact same release as
+    :term:`fMRIPrep`, please refer to TemplateFlow.
```
Comment (bthirion) [inline nilearn/datasets/struct.py:52]: Maybe add a link to TemplateFlow

File: nilearn/datasets/struct.py [hunk 2]
```diff
@@ -497,6 +510,11 @@ def fetch_icbm152_brain_gm_mask(data_dir=None, threshold=0.2, resume=True,
                                 n_iter=2, verbose=1):
     """Downloads ICBM152 template first, then loads the 'gm' mask.
 
+    The default template of :term:`fMRIPrep` is the asymmetrical ICBM152 2009,
```
Comment (NicolasGensollen) [inline nilearn/datasets/struct.py:513]: If you have the exact same message in multiple docstrings, you can consider adding it to the standard documentation in `nilearn/_utils/docs.py`

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks @htwangtw !

**Comment by NicolasGensollen** [comment]
Thanks, merging...

**Pull request merged**